### PR TITLE
[ACR-1352] Remove deprecated APIs

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Change Description ###

Describe your changes here.

- Some APIs in Rendering Effects page are deprecated from Tizen 5.5
- Removed the page and change it to Multi-threaded page
  because multi-thread is much reasonable to Rendering effects.

### Bugs Fixed ###

N/A

### API Changes ###

 - ACR: ACR-1352

### CODE REVIEW

File: docs/application/native/guides/ui/dali/animation.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can use animation to allow your objects to move around and change their prop
 
 DALi provides a rich and easy to use animation framework which allows you to create visually rich applications. The `Dali::Animation` class can be used to animate the [animatable properties](properties.md#attributes) of any number of objects.
 
-DALi animations [occur in a dedicated thread](multi-threaded.md#animations). This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
+DALi animations [occur in a dedicated thread](dali-rendering.md#animations). This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
```
Comment (divyk) [inline docs/application/native/guides/ui/dali/animation.md]: Please make a minor change in the sentence:

DALi animations [occur in a dedicated thread](dali-rendering.md#animations). It allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.

Comment (Seoyeon2Kim) [reply docs/application/native/guides/ui/dali/animation.md]: Update the file as your comment.

File: docs/application/native/guides/ui/dali/rendering-effects.md [hunk 2]
```diff
@@ -1,57 +0,0 @@
-# Rendering and Effects
```
[the remark this answers is not in the document]
Comment (Seoyeon2Kim) [reply docs/application/native/guides/ui/dali/rendering-effects.md]: As you mentioned, they're not used anymore. 
So, I'll remove the image files. :)

File: docs/application/native/guides/ui/dali/animation.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ You can use animation to allow your objects to move around and change their prop
 
 DALi provides a rich and easy to use animation framework which allows you to create visually rich applications. The `Dali::Animation` class can be used to animate the [animatable properties](properties.md#attributes) of any number of objects.
 
-DALi animations [occur in a dedicated thread](multi-threaded.md#animations). This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
+DALi animations [occur in a dedicated thread](dali-rendering.md#animations). It allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
```
[the remark this answers is not in the document]
Comment (Seoyeon2Kim) [reply docs/application/native/guides/ui/dali/animation.md]: Okay, I'll update it.

**Comment by mijin85cho** [comment]
Thanks for the fast response.
@divyk, would you please check the updated? If you think this PR is good to merge, please approve it.

**Pull request merged**